pkg/mixin/mixin.go

```diff
@@ -6,7 +6,7 @@ import (
 
 func IsCoreMixinCommand(value string) bool {
 	switch value {
-	case "install", "upgrade", "uninstall", "build", "schema", "version":
+	case "install", "upgrade", "uninstall", "build", "schema", "version", "delete":
```

Comment:
This function `IsCoreMixinCommand` should really have been documented! 😱 It is the list of commands that all mixins implement (see https://porter.sh/mixin-commands). 

It isn't the same thing as what you are implementing here `porter mixin delete` which is allowing a user to delete a specific mixin. So please remove `delete` from the case switch.